notes-app/notes.js

```diff
@@ -1,8 +1,27 @@
-const fs = require ('fs')
+const fs = require('fs')
+const chalk = require('chalk')
+
 const PATH = 'json/notes.json'
 
-const getNotes = function() {
-    return 'Your notes...'
+const readNote = function (title) {
+    const notes = loadNotes()
+    const noteToRead = notes.filter((note) => note.title === title)
+
+    if (noteToRead.length > 0) {
+        console.log(chalk.bold.green(title))
+        console.log(chalk.inverse(noteToRead[0].body))
+    } else {
+        console.log('The requested note does not exist')
+    }
+
+}
+
+const listNotes = function() {
+    const notes = loadNotes()
+    notes.forEach(element => {
+        console.log(chalk.bold.green(element.title))
+        console.log(chalk.inverse(element.body + "\n"))
+    });
```

Comment:
It would be cool if you use business language. I find element too generic

Reply:
agree!

Reply:
done